src/vs/platform/quickinput/browser/quickInputList.ts

```diff
@@ -539,7 +539,10 @@ export class QuickInputList {
 			}
 		}));
 
-		const delayer = new ThrottledDelayer(options.hoverDelegate ? options.hoverDelegate.delay : 200);
+		if (options.hoverDelegate === undefined) {
```

Comment:
there are some things at the very bottom we still need:

```
this.disposables.push(this._listElementChecked.event(_ => this.fireCheckedEvents()));
		this.disposables.push(
			this._onChangedAllVisibleChecked,
			this._onChangedCheckedCount,
			this._onChangedVisibleCount,
			this._onChangedCheckedElements,
			this._onButtonTriggered,
			this._onSeparatorButtonTriggered,
			this._onLeave,
			this._onKeyDown,
			delayer
		);
```

can you change this to like...
```
if (options.hoverDelegate) {
    // delayer creation and the two mouse events that use it
}
```